releases/index.html

```diff
@@ -23,13 +24,59 @@
   | <a href="#14.2">14.2</a>
   | <a href="#14.1">14.1</a>
 </p>
-
 <div class="grid-10">
   <div class="version">
+    <a name="16.2"></a>
     <a name="latest"></a>
-
+    <h4>16.2 (latest)</h4>
+    <div class="section">
+      <h5>What's new in GoCD 16.2</h5>
+    <div class="section">
+      <h5>Features and bug fixes</h5>
+      <ul>
+        <li>
```

Comment:
Does it make sense to link to the roadmap instead?
I'd want to cut out the noise in the release notes as you look at it with focus about new features or trouble shooting something.

The technical details of the websocket can be a blog post?